documentation/asciidoc/topics/proc_building_and_running.adoc

```diff
@@ -24,5 +24,5 @@ $ git clone {repository_ssh}
 +
 [source,bash,options="nowrap",subs=attributes+]
 ----
-$ mvn clean install -DskipTests=true
+$ mvn -s /path/to/maven-settings.xml clean install -DskipTests=true
```

Comment:
the maven-settings.xml is only required for downstream.

Reply:
thanks @pruivo I've fixed this so the maven-settings.xml arg is downstream only.